tests/jobs/test_api.py

```diff
@@ -18,6 +19,10 @@ async def test_find_beta(archived, state, fake, snapshot, spawn_client):
     if state is not None:
         url += f"&state={state}"
 
+    if users is not None:
+        for user in users.split(","):
```

Comment:
Is `split` still necessary here?

Reply:
Not sure why I kept this, I have changed the parametrization to pass in a list of users.